.github/workflows/release.yaml

```diff
@@ -5,6 +5,10 @@
 # This workflow (phase 2) creates the final release after RC testing has been completed.
 name: Release
 
+permissions:
```

Comment:
Are some of these only required for the override SDK PR https://github.com/LF-Decentralized-Trust-labs/paladin/pull/846? If so it would be good to move them there as part of that review rather than leave them here.